snowplowtracker-scm-1.rockspec

```diff
@@ -1,7 +1,8 @@
 package = "SnowplowTracker"
-version = "0.2.0-1"
+version = "scm-1"
```

Comment:
Not sure I understand the change to `scm` – what does it mean? I think it's a good idea to remove the version number from the rockspec file name. But removing it from the version string seems a bit unusual although in a quick search I have seen some Lua packages that do it. Where is the tracker version taken from when adding it as a property to events?

Reply:
The tracker version is taken from this line:
https://github.com/snowplow/snowplow-lua-tracker/blob/release/0.2.0/src/snowplow/tracker.lua#L31

Reply:
Oh, I see... If you felt like overengineering this a little bit, it'd be cool to check that the version in the `tracker.lua` file matches the current git tag before publishing, [similar to here.](https://github.com/snowplow-incubator/snowplow-roku-tracker/blob/main/.github/workflows/deploy.yml#L11-L43)

Reply:
This has been done.